ccmbioinfo/lfs
# CARDS-2100: Use the encounter identifier as the visit subject identifier

To test:
- build with `-Pdocker`
- in `compose-cluster` run `python3 generate_compose_yaml.py --mssql --cards_project cards4prems --oak_filesystem --dev_docker_image --smtps --smtps_test_container --smtps_test_mail_path ~/cardsmail --composum --adminer`
- start with `docker-compose build && docker-compose up`
- at `http://localhost:1435` log in and import `compose-cluster/mssql/processorImportTestSql.sql`
- trigger an import at `localhost:8080/Subjects.importClarity?pastDayToQuery=1`
- check that visits now have short `20xx` identifiers instead of long random UUIDs
